doc/introduction/chemistry.rst

```diff
@@ -0,0 +1,192 @@
+using the electronic structure packages `PySCF <https://github.com/sunqm/pyscf>`_ and `Psi4
+<http://psicode.org/>`_, respectively.
+
+For this, it is required to specify a string to label the molecule, its net charge, the `spin
+multiplicity <https://en.wikipedia.org/wiki/Multiplicity_(chemistry)>`_ and the  `atomic basis
+functions <https://www.basissetexchange.org/>`_.
+
+.. code-block:: python
+
+    geometry = qml.qchem.read_structure('h2o.SDF')
+    hf_data = qml.qchem.gen_meanfield_data(
+        'water',
+        geometry,
+        charge=0,
+        multiplicity=1,
+        basis_set='sto-3g',
+        qc_package='pyscf'
+    )
+
+The output variable ``hf_data`` stores the path to the directory containing the file ``'water
+.hdf5'`` with the Hartree-Fock electronic structure of the water molecule.
+
+Mapping the Hamiltonian to the Pauli basis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The function :func:`~.gen_active_space` is used to create an `active space <https://en.wikipedia
+.org/wiki/Complete_active_space>`__  by classifying the Hartree-Fock molecular orbitals as doubly-occupied,
+active and external orbitals. Within this approximation, a certain number of *active electrons*
+can populate the *active orbitals*.
+
+.. code-block:: python
+
+    d_occ_indices, active_indices = qml.qchem.gen_active_space(
+        'water',
+        hf_data,
+        n_active_electrons=2,
+        n_active_orbitals=2
+    )
+
+Once we have defined the active space, :func:`~.gen_hamiltonian_pauli_basis` calls
+OpenFermion to generate the second-quantized fermionic Hamiltonian
+and map it to a linear combination of Pauli operators via the `Jordan-Wigner
+<https://en.wikipedia.org/wiki/Jordan%E2%80%93Wigner_transformation>`__ or `Bravyi-Kitaev
+<https://arxiv.org/abs/1208.5986>`__ transformation. For example,
+
+.. code-block:: python
+
+    qubit_hamiltonian = qchem.gen_hamiltonian_pauli_basis(
+        'water',
+        hf_data,
+        mapping='jordan_wigner',
+        docc_mo_indices=d_occ_indices,
+        active_mo_indices=active_indices
+    )
+
+Here, ``qubit_hamiltonian`` is an instance of the QubitOperator class of `OpenFermion <https://github
+.com/quantumlib/OpenFermion>`_.
+
+.. _pl_qchem_vqe:
+
+VQE simulations
+---------------
+
+The Variational Quantum Eigensolver (VQE) is a hybrid quantum-classical computational scheme,
+where a quantum computer is used to prepare the trial wave function of a molecule and to measure
+the expectation value of the *electronic Hamiltonian*, while a classical optimizer is used to
+find the lowest eigenvalue of the measured Hamiltonian.
+
+First, the qubit Hamiltonian ``h`` has to be converted to a PennyLane
+Hamiltonian using the :func:`~.load_hamiltonian` function:
+
+.. code-block:: python
+
+    hamiltonian = qml.qchem.load_hamiltonian(h)
+
+where ``hamiltonian`` is 2-tuple of coefficients and PennyLane Pauli observables.

```

Comment:
``hamiltonian`` is a ``pennylane.beta.vqe.Hamiltonian`` object which is represented as a 2-tuple internally. It's perhaps not necessary to be very precise in the quickstart, but I can see it causing confusion from reading this line.

Reply:
Maybe we should just remove this sentence?

Reply:
I think that works!

Reply:
done!